devicetypes/smartthings/aeon-key-fob.src/aeon-key-fob.groovy

```diff
@@ -80,16 +81,22 @@ def zwaveEvent(physicalgraph.zwave.commands.wakeupv1.WakeUpNotification cmd) {
 
 def buttonEvent(button, held) {
 	button = button as Integer
-	String childDni = "${device.deviceNetworkId}/${button}"
-	def child = childDevices.find{it.deviceNetworkId == childDni}
-	if (!child) {
-		log.error "Child device $childDni not found"
+	def child
+	if(state.buttons==4)
```

Comment:
It's a small point but can we make this `if (state.buttons > 1)`? That way if they come out with a two or six button device it will work as-is